shadow.c

```diff
@@ -117,10 +117,19 @@ static PHP_GSHUTDOWN_FUNCTION(shadow)
 }
 /* }}} */
 
+/* {{{ shadow_deps
+ */
+static const zend_module_dep shadow_deps[] = {
+	ZEND_MOD_OPTIONAL("zip")
```

Comment:
I wouldn't mention any particular extensions here, since it would solve the problem for the mentioned ones and create a false feeling of the problem having been fixed, but for the rest of extensions it would still exist.